doc/installation_windows.md

```diff
@@ -1,18 +1,13 @@
-# Windows 8.1 & Windows 10 Installation
+# Windows 10 & Windows 11 Installation
 
 **Note:** Due to the USB 3.0 translation layer between native hardware and virtual machine, the librealsense team does not recommend or support installation in a VM.
 
-librealsense shall be built on Windows using [CMake v3.8+](https://cmake.org/download/) and Visual Studio 2015 / 2017: 
+librealsense shall be built on Windows using [CMake v3.8+](https://cmake.org/download/) and Visual Studio 2017 / 2019 / 2022: 
 (MSVC2013 and older are not fully compatible with the C++11 feature-set).
```

Comment:
This should really be mended to 2015 and older with the C++14 feature-set

Reply:
I will fix it soon, please not merge.